src/main/java/net/snowflake/ingest/connection/RequestBuilder.java

```diff
@@ -61,6 +62,9 @@ public class RequestBuilder {
   // Reference to the telemetry service
   private final TelemetryService telemetryService;
 
+  // The authorization type used in auth header
+  public final String authType;
```

Comment:
Could you infer this from the SecurityManager type?

Reply:
Yes.